src/bridges/chainlink-ccip/CcipGhoBridge.sol

```diff
@@ -0,0 +1,177 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import {Ownable} from 'solidity-utils/contracts/oz-common/Ownable.sol';
+import {IERC20} from 'solidity-utils/contracts/oz-common/interfaces/IERC20.sol';
+import {LinkTokenInterface} from '@chainlink/contracts/src/v0.8/shared/interfaces/LinkTokenInterface.sol';
+import {CCIPReceiver} from '@chainlink/contracts-ccip/src/v0.8/ccip/applications/CCIPReceiver.sol';
+import {IRouterClient} from '@chainlink/contracts-ccip/src/v0.8/ccip/interfaces/IRouterClient.sol';
+import {Client} from '@chainlink/contracts-ccip/src/v0.8/ccip/libraries/Client.sol';
+
+import {ICcipGhoBridge} from './ICcipGhoBridge.sol';
+
+/**
+ * @title CcipGhoBridge
+ * @author LucasWongC
+ * @notice Helper contract to bridge GHO using Chainlink CCIP
+ */
+contract CcipGhoBridge is ICcipGhoBridge, CCIPReceiver, Ownable {
+  /// @dev Chainlink CCIP router address
+  address public immutable ROUTER;
+  /// @dev LINK token address
+  address public immutable LINK;
+  /// @dev GHO token address
+  address public immutable GHO;
+
+  /// @dev Address of bridge (chainSelector => bridge address)
+  mapping(uint64 => address) public bridges;
```

Comment:
You can name the variables:

`mapping (uint64 selector => address bridge) public bridges`